Doc/whatsnew/3.11.rst

```diff
@@ -126,6 +126,11 @@ Deprecated
 Removed
 =======
 
+* The :func:`@asyncio.coroutine <asyncio.coroutine>` :term:`decorator` enabling
+  legacy generator-based coroutines to be compatible with async/await code.
+  The function has been deprecated since Python 3.8 and the removal was
+  initially scheduled for Python 3.10. Use :keyword:`async def` instead.
+  (Contributed by Illia Volochii in :issue:`43216`.)
```

Comment:
Since CoroWrapper name is public, please document also its removal. (here and in the NEWS entry)

Reply:
Done 👌🏻